integrations/docinfo.xml

```diff
@@ -0,0 +1,13 @@
+<title>Integrations</title>
+<productname>{product-title}</productname>
+<productnumber>{product-version}</productnumber>
+<subtitle>Integrating OpenShift Serverless with Service Mesh and with the cost management service</subtitle>
```

Comment:
You know better than I do, but please double-check the capitalization of "cost management service."